list.txt

```diff
@@ -22,3 +22,144 @@ zoom,
 teams,
 database,
 server,
+AI,
+Blue Peter,
+Bob the Builder,
+Game of Thrones,
+Raven,
+Doctor Who,
+Jungle Run,
+My Parents are Aliens,
+ChuckleVision,
+Art Attack
+Thomas the Tank Engine,
+Discworld,
+The Laundry Files,
+Star Trek,
+Star Wars,
+Monty Python,
+Fawlty Towers,
+Robin Hood,
+King Arthur,
+Merlin,
+Guy Fawkes,
+Garden of Eden,
+10 Downing Street,
+Buckingham Palace,
+Holyrood,
```

Comment:
Hollywood

Reply:
Actually, this was going for the nickname of the Scottish parliament (which I've double checked has a single l).

Reply:
Oh! I didn't know that!